sys/linux/test/syz_image_mount_f2fs_9

```diff
@@ -0,0 +1 @@
+syz_mount_image$f2fs(&(0x7f0000000000)='f2fs\x00', &(0x7f0000000100)='./file0\x00', 0x4000000, 0x0, &(0x7f0000000200)=[], 0x0, &(0x7f0000010000))
```

Comment:
What happened to all of these tests?  They are missing the images now...

Reply:
Seems to be cause by the white spaces and empty string in the options. Just fixed it (although not exactly sure why it works now)